.github/workflows/build_cmake.yml

```diff
@@ -123,3 +123,13 @@ jobs:
         name: ni-grpc-device-server-linux-glibc2_23
         path: ni-grpc-device-server-linux-glibc2_23.tar
         retention-days: 5
+
+    - name: Upload Windows Server Binaries Artifact
+      uses: actions/upload-artifact@v2
+      if: "(github.ref == 'refs/heads/main') && (runner.os == 'Windows')"
```

Comment:
This should also run on `refs/heads/releases/*`

Reply:
I used `startsWith` as it wasn't clear from the documentation if operators accepted wildcard filters.